src/debugAdapter/runtimeInterface.ts

```diff
@@ -194,28 +201,6 @@ export default class RuntimeInterface extends EventEmitter {
     }
   }
 
-  /**
-   * Retrieves the chain id of the `providerUrl`.
-   *
-   * @param providerUrl the url to get chain id from.
-   * @returns the chain id of the given `providerUrl`.
-   */
-  private getNetworkId(providerUrl: string) {
-    const services = TreeManager.getItem(ItemType.LOCAL_SERVICE);
-
-    if (!services || !services.getChildren()) {
-      return undefined;
-    }
-
-    const projects = services.getChildren() as LocalProject[];
-    const project = projects.find((project) => {
-      const network = project.getChildren().at(0) as LocalNetworkNode;
-      return `${network.url.protocol}//${network.url.host}` === providerUrl;
-    });
-
-    return project ? getChainId(project.options.forkedNetwork) : undefined;
-  }
-
```

Comment:
This is great! By removing all the `TreeManager` related code within the Debugger we make it less brittle.